submissions/akinbobolaFaith/dom/signUp.js

```diff
@@ -0,0 +1,122 @@
+let signupForm = document.querySelector('.information'),
+    firstName = document.getElementById('fname'),
+    lastName = document.getElementById('lname'),
+    phoneNumber = document.getElementById('pnumber'),
+    email = document.getElementById('email'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('cpassword')
+;
+
+function important (field, event) {
+    if(field.value == '') {
+        field.nextElementSibling.innerHTML = "This field is required";
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+function firstNameValidation(event) {
```

Comment:
First  name validation and last name validation are practically the same thing. I think you should pass in the innerHTML as a variable and use just one function @faithakinbobola. Also, you can pass the firstName and lastName as a variable.